### PR TITLE
Integration testing data for preprocessing

### PR DESCRIPTION
**Pull request by avramidis**

### CODE REVIEW

**Comment by karlnapf** [comment]
Nice! Now to trigger the ci you need to update the data submodule in the develop branch pr
Once that’s green we can merge first data and then the examples

**Comment by karlnapf** [comment]
We will merge this just before the other PR to keep the CI happy

File: testsuite/meta/preprocessor/fishlda.dat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<<_SHOGUN_SERIALIZABLE_ASCII_FILE_V_00_>>
+array Vector<SGSerializable*> 0 ()
+resize_granularity int32 128
+use_sg_malloc bool t
+free_array bool t
+dim1_size int32 1
+dim2_size int32 1
+dim3_size int32 1
```
Comment (karlnapf) [inline testsuite/meta/preprocessor/fishlda.dat]: All these are empty, just realised

**Comment by avramidis** [comment]
I will have one pass today and tomorrow and fix these.

**Pull request merged**